packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -1053,7 +1066,7 @@ RED.palette.editor = (function() {
         const sortRelevance = $('<a href="#" class="red-ui-palette-editor-install-sort-option red-ui-sidebar-header-button-toggle selected"><i class="fa fa-sort-amount-desc"></i></a>').appendTo(sortGroup);
         const sortAZ = $('<a href="#" class="red-ui-palette-editor-install-sort-option red-ui-sidebar-header-button-toggle"><i class="fa fa-sort-alpha-asc"></i></a>').appendTo(sortGroup);
         const sortRecent = $('<a href="#" class="red-ui-palette-editor-install-sort-option red-ui-sidebar-header-button-toggle"><i class="fa fa-calendar"></i></a>').appendTo(sortGroup);
-        RED.popover.tooltip(sortRelevance,RED._("palette.editor.sortRelevance"));
+        RED.popover.tooltip(sortRelevance,RED._("palette.editor.sortDownloads"));
```

Comment:
```suggestion
        RED.popover.tooltip(sortRelevance,RED._("palette.editor.sortRelevance"));
```
Keep the 'relevance' label; we're using downloads to help determine relevance.